apps/chat-e2e/src/testData/api/itemApiHelper.ts

```diff
@@ -5,29 +5,40 @@ import { API } from '@/src/testData';
 import { BaseApiHelper } from '@/src/testData/api/baseApiHelper';
 import { BucketUtil, ItemUtil } from '@/src/utils';
 import { expect } from '@playwright/test';
+import * as process from 'node:process';
 
 export class ItemApiHelper extends BaseApiHelper {
-  public async deleteAllData(bucket?: string) {
-    const conversations = await this.listItems(API.conversationsHost(), bucket);
-    const prompts = await this.listItems(API.promptsHost(), bucket);
-    await this.deleteBackendItem(...conversations, ...prompts);
+  public async deleteAllData(isOverlay?: boolean, bucket?: string) {
```

Comment:
it would be better change argument places to not pass `false` each time, but only `true` value
```suggestion
  public async deleteAllData(bucket?: string, isOverlay = false) {
```

Reply:
thank you, fixed